contrib/DEBIAN/copyright

```diff
@@ -77,6 +78,331 @@ License: PUB-DOM
 Copyright: Crobbo (forum)
 Comment: Site: https://bitcointalk.org/index.php?topic=32273.0
 
+Files: depends/sources/libsodium-*.tar.gz
+Copyright: 2013-2016 Frank Denis
+License: ISC
+
+Files: depends/sources/libsnark-*.tar.gz
+Copyright: 2012-2016 SCIPR Lab and contributors; 2016 Zcash developers
```

Comment:
+1, but in that case we should change https://github.com/zcash/zcash/blob/master/COPYING#L1 .